### PR TITLE
Install script for Binance Chain node

### PR DESCRIPTION
**Pull request by onggunhao**
# Summary

* `install.sh` for autoinstaller script (verified for Mac and *nix, but not Windows)
* Updated README

# Spec
We want a simplified install script for Binance Chain nodes, that great simplifies the experience. This is both for lightnodes and full nodes. This makes it easy for people to get started during in-person workshops.

This is inspired by Docker's https://get.docker.com, a shell script that takes care of all setup for Docker

Longer-term, we should explore distributing the node binaries through a package manager like `brew` and `apt` for Mac and Linux respectively (see related issue #148)

### CODE REVIEW

**Comment by onggunhao** [comment]
@yutianwu @guagualvcha Btw - please note that the install script is **fragile** because we use folder names to hold versions. So a wrong rename (e.g. "lightd" instead of "lightnode") can result in a 404 during `wget` 

I think this can be fixed by tracking versions using branches, instead of folders. Similar to Bitcoin repo. I created issue here https://github.com/binance-chain/node-binary/issues/144. 

Once we do that, we can create a package installation using `brew` or `apt-get`. Issue created: https://github.com/binance-chain/node-binary/issues/144

**Comment by huangsuyu** [comment]
hello, @onggunhao 

Thanks for your PR. 

I think it's better that users can customized `BNC_HOME_CONFIG_DIR `.

Can you add it as extrq step please?

Thanks!

**Comment by onggunhao** [comment]
That makes sense, I'll add it

File: README.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+## Installation Script
+
+We have a community-maintained installer script (`install.sh`) that takes care of chain directory setup. This uses the following defaults:
+
+* Home folder in `~/.bnbchaind`
+* Client executables stored in `/usr/local/bin` (i.e. `light` or `bnbchaind`)
+
+```shell
+# One-line install
+sh <(wget -qO- https://raw.githubusercontent.com/onggunhao/node-binary/master/install.sh)
```
[the remark this answers is not in the document]
Comment (huangsuyu) [reply README.md]: Yes, it's working for me!

**Comment by varnav** [comment]
Also there is a docker image available for simplified setups:

https://github.com/varnav/binance-node-docker

**Comment by huangsuyu** [comment]
@onggunhao 

Hello, Dan! I add the command for choosing `BNC_HOME_DIR`. And I think it's better to choose the network type first.

**Comment by huangsuyu** [comment]
> https://github.com/varnav/binance-node-docker

I will 

> Also there is a docker image available for simplified setups:
> 
> https://github.com/varnav/binance-node-docker

I'd love to add this in README too.

**Pull request merged**